.github/workflows/lint.yaml

```diff
@@ -56,5 +56,4 @@ jobs:
           sudo apt-get install -y libvirt-dev
       - name: check
         run: make CLOUD_PROVIDER=${{ matrix.provider }} check
-      - name: escapes
```

Comment:
in discussion phase, I suppose we can enable this job to see different between others.